recipe/patches/0001-Blacklist-knl-for-clang-19.patch

```diff
@@ -0,0 +1,38 @@
+From 620c17041007fd73294d01044cd1138c4b8faead Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@quansight.com>
+Date: Mon, 10 Nov 2025 19:37:14 +0100
+Subject: [PATCH] Blacklist knl for clang 19+
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Fixes #907
```

Comment:
Not sure how this happened, but your commits upstream had a lot more context on this, i.e. the commit message in https://github.com/flame/blis/commit/071dfce5f7b1100bedbe2e1704b0f7c65639cce4 (or the equivalent commit in the PR itself) is not reflected here. Could you add that please?